deltachat-ios/Controller/GroupChatDetailViewController.swift

```diff
@@ -237,7 +244,12 @@ class GroupChatDetailViewController: UIViewController {
     }
 
     private func showQrCodeInvite(chatId: Int) {
-        let qrInviteCodeController = QrInviteViewController(dcContext: dcContext, chatId: chatId)
+        var hint = ""
+        let dcChat = dcContext.getChat(chatId: chatId)
+        if !dcChat.name.isEmpty {
+            hint = String.localizedStringWithFormat(String.localized("qrshow_join_group_hint"), dcChat.name)
+        }
+        let qrInviteCodeController = QrViewController(dcContext: dcContext, chatId: chatId, qrCodeHint: hint)
```

Comment:
I refactored QRViewController in order to be able to dump QRInviteViewController, which was hard to adjust for dynamic font sizes. Now the hint below the QR code has to be injected.